src/geophires_x/AGSWellBores.py

```diff
@@ -1078,7 +1102,10 @@ def Calculate(self, model: Model) -> None:
             self.ProducedTemperature.value = self.InterpolatedTemperatureArray.copy()
 
             tot_length, vert_length, horizontal_lengths = self.calculatedrillinglengths(model)
-            model.reserv.depth.value = model.reserv.InputDepth.value * 1000.0  # in this case, reserv.depth is just the vertical drill depth
+
+            # in this case, reserv.depth is just the vertical drill depth
+            # FIXME earlier calculations use depth before this value is set, meaning they're using the wrong value
```

Comment:
Created follow-up issue: https://github.com/NREL/GEOPHIRES-X/issues/122

Reply:
Moved comment reply from email to https://github.com/NREL/GEOPHIRES-X/issues/122#issuecomment-1948693134